src/components/EvolutionTab.js

```diff
@@ -128,7 +130,21 @@ const EvolutionTab = ({ evolutionChainUrl }) => {
     return organizedArray;
   };
 
-  return <div className="spriteContainer">{spritesWithEvolutionArrows()}</div>;
+  return (
+    <>
+      <div
+        style={{
+          // custom scrollbar (firefox only)
+          scrollbarColor: `${pokemonColor} rgba(235, 232, 232, 0.4)`,
+          overflowX: "auto"
+          // ** unknown solution for safari and chrome **
```

Comment:
stinks that it doesn't work on there

Reply:
I lucked out since i like to use firefox as my main browser 👍